File: pm_core/src/core/domain/entity/priority_scheme_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type PrioritySchemeEntity struct {
	BaseEntity
	TenantID          int64   `json:"tenantId"`
	Name              string  `json:"name"`
	Description       *string `json:"description,omitempty"`
	DefaultPriorityID int64   `json:"defaultPriorityId"`

	Items []PrioritySchemeItemEntity `json:"items,omitempty"`
}

File: pm_core/src/main/java/serp/project/pmcore/core/domain/entity/PrioritySchemeEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.core.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class PrioritySchemeEntity extends BaseEntity {
    private Long tenantId;
    private String name;
    private String description;
    private Long defaultPriorityId;

    private List<PrioritySchemeItemEntity> items;
}
